docs/topics/native/native-cocoapods-libraries.md

```diff
@@ -285,4 +285,28 @@ kotlin {
         }
     }
 }
-```
\ No newline at end of file
+```
+
+### Sharing Kotlin cinterop between dependent Pods
+
+If you're adding multiple dependencies on Pods using the `pod()` function, you might encounter issues in
+case there are dependencies between APIs of your Pods.
```

Comment:
in case of?
or better to add another if?
or

If you add multiple dependencies on Pods using the `pod()` function, you might face issues when there are dependencies between the APIs of your Pods.